led_controller:
    type: cooperative_driving/LedController
    publish_rate: 50
    cmd_timeout: .5
